### DISCUSSION TITLE
Use a SSH Profile In Git Extensions?

### DISCUSSION THREAD

**Discussion by ThePieMonster** [gitextensions/gitextensions#11632] [Q&A]
Is there a way to set a .bashrc and .bash_profile via Git Extensions? I have a home path set with these two files in the root dir. 

When using Git Bash a script runs on first open which ensures the ssh client is running and thus does not re-prompt for a passphrase over and over again. 

Anyway to use a profile in Git Extensions? 

<img width="410" alt="Screenshot 2024-03-11 133900" src="https://github.com/gitextensions/gitextensions/assets/11359134/5b238a1b-d3bd-4705-9178-770a5c1cc62c">

Environment is the same as described in this issue.
https://github.com/gitextensions/gitextensions/issues/10654

**Comment by RussKie** [comment]
No, the app doesn't provide such functionality.

**Comment by gerhardol** [comment]
Use PuTTY integration
